src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -16,14 +16,31 @@
 public class HelpWindow extends UiPart<Stage> {
 
     public static final String USERGUIDE_URL = "https://ay2021s1-cs2103t-t12-4.github.io/tp/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+
+    public static final String COMMON_COMMANDS = "Add a patient ---"
+            + "add n/NAME i/NRIC p/PHONE_NUMBER a/ADDRESS [t/TAG]…\n"
+            + "Edit a patient --- edit INDEX [n/NAME] [i/NRIC] [p/PHONE_NUMBER] [a/ADDRESS] [t/TAG]…\n"
+            + "Delete a patient --- delete INDEX\n"
+            + "List all patients --- list\n\n"
+            + "Assign a patient to an appointment --- assign INDEX d/DATE t/TIME [dur/DURATION]\n"
+            + "Change an appointment time --- change INDEX [d/DATE] [t/TIME] [dur/DURATION]\n"
+            + "Cancel an appointment --- cancel d/DATE t/TIME\n"
+            + "Mark an appointment as done --- done d/DATE t/TIME\n"
+            + "Find available time slots --- avail d/DATE\n"
+            + "View appointments by date --- view [d/DATE]\n\n"
+            + "Undo a previous command --- undo\n"
+            + "Redo a previous command --- redo\n";
```

Comment:
Perhaps a line showcasing the remark command can be included as well? Let me know what you think 😄

Reply:
Ok added